src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss

```diff
@@ -0,0 +1,98 @@
+.container {
```

Comment:
let's not rewrite the bootstrap class, but use another one

Reply:
fixed